src/iuse.cpp

```diff
@@ -4332,15 +4278,25 @@ int iuse::circsaw_on(player *p, item *it, bool t, const tripoint& )
 
 int iuse::jackhammer(player *p, item *it, bool, const tripoint &pos )
 {
+    bool normal_language = it->type->id != "jacqueshammer";
+      // Jacqueshammers function the same as ordinary
+      // jackhammers, except they print messages in French for
+      // comic effect.
     if (it->charges < it->type->charges_to_use()) {
         return 0;
     }
     if (p->is_underwater()) {
-        p->add_msg_if_player(m_info, _("You can't do that while underwater."));
+        p->add_msg_if_player(m_info, normal_language
+          ? _("You can't do that while underwater.")
+          : _("Vous ne pouvez pas faire que sous l'eau."));
```

Comment:
the comments that look like ```//~ (jacqueshammer) "Drill where?"``` are magic comments that get attached to the next translatable string for use by our hardworking translators.  The Jaqueshammer messages in particular need them because people get confused by there being french scattered around.